torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1946,8 +1941,7 @@ def sample_inputs_addmm(op_info, device, dtype, requires_grad, **kwargs):
         sample_inputs.append(
             SampleInput(make_tensor(shape, device, dtype, requires_grad=requires_grad),
                         args=(
-                            make_tensor(shape, device, dtype,
-                                        requires_grad=requires_grad).t().conj(),
+                            make_tensor(shape, device, dtype).mH.requires_grad_(requires_grad),
```

Comment:
do you get an error if you don't set the requires grad? I know you left a note above to set requires grad after performing any transformations but I am worried because it's an easy thing to miss

Reply:
Yes, you do get an error. I needed to update all these functions for the tests to pass.

Reply:
maybe we should create helper functions for standard transformations that people can use. We can take care of retaining the `requires_grad` field in those helper functions then!